alf/examples/ac_target_navigation_attn.gin

```diff
@@ -1,6 +1,6 @@
 include 'ac_target_navigation.gin'
 
-target_navigation.get_ac_networks.attention=True
+# target_navigation.get_ac_networks.attention=True
```

Comment:
why commented out this?

Reply:
@witwolf the gin configs are out of sync from the target_navigation.gin.  If needed, I could submit the newest version for review.  Easiest might be to just exclude them from the tests if they are causing trouble.  Let me know what you prefer.

Thanks,
Le

Reply:
@le-horizon  you can sync them after this pr

Reply:
@witwolf you can restore the file and remove it from the test temporarily. And assign an issue to @le-horizon to fix it.

Reply:
removed temporarily

Reply:
Could you revert this change, and you may need to exclude this gin from tests as well.  Thanks @witwolf !